package.json

```diff
@@ -33,9 +33,10 @@
     "@babel/preset-env": "^7.12.11",
     "@babel/preset-react": "^7.12.10",
     "@enact/dev-utils": "^4.0.0",
-    "@storybook/addon-actions": "6.2.9",
-    "@storybook/addon-docs": "6.2.9",
-    "@storybook/addon-knobs": "6.2.9",
+    "@storybook/addon-actions": "^6.3.7",
+    "@storybook/addon-controls": "^6.3.7",
+    "@storybook/addon-docs": "^6.3.7",
+    "@storybook/addon-toolbars": "^6.3.7",
     "babel-plugin-dev-expression": "^0.2.2",
     "less": "^3.13.0",
     "less-loader": "^6.2.0",
```

Comment:
Please update @storybook-related modules in devDependencies.

Reply:
OK

Reply:
OK